api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -416,6 +422,33 @@ private FirecloudWorkspaceId generateFirecloudWorkspaceId(String namespace, Stri
             dbWorkspace.getWorkspaceNamespace(), dbWorkspace.getWorkspaceId(), dbCohort.getCohortId()));
       }
     }
+
+    List<JSONObject> demoConceptSets = cloudStorageService.readAllDemoConceptSets();
+    for (JSONObject conceptSet: demoConceptSets) {
+      ConceptSet dbConceptSet = new ConceptSet();
+      JSONArray conceptIdsJSON = conceptSet.getJSONArray("concept_ids");
```

Comment:
This can throw `JSONException` ... should catch this

Reply:
how does that work?  am confused about syntax

Reply:
Something like this
```suggestion
      try {
        JSONArray conceptIdsJSON = conceptSet.getJSONArray("concept_ids");
      } catch (JSONException e) {
        // log exception
        // return an internal server error 
      }
```